pkg/cli/command_test.go

```diff
@@ -2,580 +2,390 @@ package cli
-			args: []string{
-				"list",
-				"--dirpath",
-				filepath.Join(getTestsDir(), "bad_yamls", "document_with_syntax_error.yaml"),
-			},
-			expectedOutput: "0.0.0.0-255.255.255.255 => default/checkoutservice[Deployment] : All Connections\n" +
-				"0.0.0.0-255.255.255.255 => default/emailservice[Deployment] : All Connections\n" +
-				"0.0.0.0-255.255.255.255 => default/recommendationservice[Deployment] : All Connections\n" +
-				"default/checkoutservice[Deployment] => 0.0.0.0-255.255.255.255 : All Connections\n" +
-				"default/checkoutservice[Deployment] => default/emailservice[Deployment] : All Connections\n" +
-				"default/checkoutservice[Deployment] => default/recommendationservice[Deployment] : All Connections\n" +
-				"default/emailservice[Deployment] => 0.0.0.0-255.255.255.255 : All Connections\n" +
-				"default/emailservice[Deployment] => default/checkoutservice[Deployment] : All Connections\n" +
-				"default/emailservice[Deployment] => default/recommendationservice[Deployment] : All Connections\n" +
-				"default/recommendationservice[Deployment] => 0.0.0.0-255.255.255.255 : All Connections\n" +
-				"default/recommendationservice[Deployment] => default/checkoutservice[Deployment] : All Connections\n" +
-				"default/recommendationservice[Deployment] => default/emailservice[Deployment] : All Connections",
-			exact: true,
-			isErr: false,
+			// when format is empty - output should be in defaultFormat (txt)
+			dir1: "onlineboutique",
+			dir2: "onlineboutique_with_pods_severe_error",
 		},
 		{
-			name: "test_list_dir_with_severe_error_running_with_fail_stops_and_return_empty_output",
-			// as we saw in a previous test on same path, when --fail is not used, the test produces connectivity map
-			args: []string{
-				"list",
-				"--dirpath",
-				filepath.Join(getTestsDir(), "bad_yamls", "document_with_syntax_error.yaml"),
-				"--fail",
-			},
-			expectedOutput: "found character that cannot start any token",
-			exact:          false,
-			isErr:          true, // fatal error because err is issued by the builder and uses stopOnErr
+			dir1:       "onlineboutique_workloads",
+			dir2:       "onlineboutique_workloads_changed_workloads",
+			format:     outpkg.TextFormat,
+			outputFile: outFileName,
 		},
+	}
+	for _, tt := range cases {
+		tt := tt
+		testName := testutils.DiffTestName(tt.dir1, tt.dir2) + "_format_" + determineFileSuffix(tt.format)
+		t.Run(testName, func(t *testing.T) {
+			args := []string{"diff", "--dir1", filepath.Join(testutils.GetTestsDirWithDepth(currentDirDepth), tt.dir1), "--dir2",
+				filepath.Join(testutils.GetTestsDirWithDepth(currentDirDepth), tt.dir2)}
+			args = append(args, addCmdOptionalArgs(tt.format, tt.outputFile, "")...)
+			actualOut, err := buildAndExecuteCommand(args)
+			require.Nil(t, err, "test: %q", testName)
+			expectedOutputFileName := strings.Replace(testName, testutils.FormatStr, testutils.DotSign, 1)
```

Comment:
add the generation of expected output file name to the function `DiffTestName`, similar to the `connlist` case

Reply:
done